connection_timeout: 10.0
action_timeout: 300.0
state_timeout: 0.5
command_timeout: 0.5

waypoint_dist_tolerance: 1.20
final_dist_tolerance: 0.05
waypoint_yaw_tolerance: 1.55
final_yaw_tolerance: 0.35
waypoint_time_in_tolerance: 0.0
final_time_in_tolerance: 1.0
action_frequency: 25

landing_height: 0.1
takeoff_height: 0.1

estop_deceleration: 1.0

wrench_limits:
    force: # N
        z:
            min: 0.0
            max: 30.0
    torque: # Nm
        x:
            max: 10.0
        y:
            max: 10.0
        z:
            max: 1.0
